package.json

```diff
@@ -39,6 +39,7 @@
     "@web/dev-server-import-maps": "^0.0.6",
     "@web/test-runner": "0.13.27",
     "@web/test-runner-commands": "0.6.1",
+    "@adobe/helix-rum-js": "1.6.0",
```

Comment:
This seems to be an omission, we want to pull this from the public npm instance, correct? https://github.com/adobe/helix-rum-js

Reply:
Currently this points to artifcatory, which is definitely nothing we want to include in the setup

Reply:
you are right. 
I have updated it